tests/test_registration_and_login.py

```diff
@@ -0,0 +1,180 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+
+# Import the application and database setup
+from hushline import create_app, db
+from hushline.generate_invite_codes import create_invite_code
+
+# Import models and other modules
+from hushline.model import User
+
+
+@pytest.fixture
```

Comment:
all fixtures should have `(scope='function')` unless there's a compelling reason not to and be in `tests/conftest.py`

Reply:
👍